StructureDefinition/England-Organization.xml

```diff
@@ -0,0 +1,171 @@
+  <copyright value="Copyright © 2021+ HL7 UK Licensed under the Apache License, Version 2.0 (the \&quot;License\&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \&quot;AS IS\&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7® FHIR® standard Copyright © 2011+ HL7 The HL7® FHIR® standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html." />
+  <fhirVersion value="4.0.1" />
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="Organization" />
+  <baseDefinition value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Organization" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Organization.extension">
+      <path value="Organization.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="0" />
+    </element>
+    <element id="Organization.extension:relationships">
+      <path value="Organization.extension" />
+      <sliceName value="relationships" />
+      <min value="0" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="https://fhir.nhs.uk/StructureDefinition/Extension-England-ODSOrganisationRelationships" />
+      </type>
+      <isModifier value="false" />
+    </element>
+    <element id="Organization.identifier">
+      <path value="Organization.identifier" />
+      <definition value="**MUST** contain an ANANA/ODS code. &#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA; &quot;identifier&quot;:  [&#xD;&#xA;        {&#xD;&#xA;            &quot;use&quot;: &quot;official&quot;,&#xD;&#xA;            &quot;system&quot;: &quot;https://fhir.nhs.uk/Id/ods-organization-code&quot;,&#xD;&#xA;            &quot;value&quot;: &quot;RR805&quot;&#xD;&#xA;        }&#xD;&#xA;    ],&#xD;&#xA;```" />
+      <comment value="MUST contain an ANANA/ODS code." />
+      <min value="1" />
+      <mustSupport value="true" />
+    </element>
+    <element id="Organization.identifier:odsOrganisationCode">
+      <path value="Organization.identifier" />
+      <sliceName value="odsOrganisationCode" />
+    </element>
+    <element id="Organization.identifier:odsOrganisationCode.assigner">
+      <path value="Organization.identifier.assigner" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Organization" />
```

Comment:
As per discussion, should not be adding references

Reply:
Pushed change